versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -9,7 +9,7 @@ id: releases
 May 6, 2022
 
 ### Bugfixes
```

Comment:
can we remove the 0.11 notes from here. This is the version for 0.10

Reply:
I've removed my addition along with the 0.11 notes from April.